pkg/render/goldmane/component.go

```diff
@@ -15,8 +15,11 @@
 package goldmane
 
 import (
+	"encoding/json"
 	"fmt"
+	"path/filepath"
 
+	"github.com/sirupsen/logrus"
```

Comment:
Were you intending to introduce logrus to the operator?

Reply:
nope